Cpp/odin-views/app.cpp

```diff
@@ -136,6 +136,16 @@ namespace {
             fostlib::pg::connection cnx{fostgres::connection(config, req)};
             auto ref = odin::reference();
             odin::create_user(cnx, ref, fostlib::coerce<fostlib::string>(fed_response_data["identity"]["id"]));
+            if ( odin::is_module_enabled(cnx, "opts/full-name") ){
```

Comment:
One API call and SQL request per module? .... Seems a bit much. OK for v1, but should be better in the future.